src/util/NuulsUploader.hpp

```diff
@@ -0,0 +1,18 @@
+#include "common/Channel.hpp"
+#include "widgets/helper/ResizingTextEdit.hpp"
+
+#include <QMimeData>
+#include <QString>
+
+namespace chatterino {
+struct TypedBytes {
```

Comment:
`TypedBytes` is not a particularly descriptive class name, when what it's actually for is grouping together image data and its format. Try to figure out a better name for it if possible.
Also, `format` is a better description of the data instead of `type`.